javascript-local-storage/data.js

```diff
@@ -2,7 +2,10 @@
 
 var todos = [];
 var previousTodosJSON = localStorage.getItem('javascript-local-storage');
-todos = JSON.parse(previousTodosJSON);
+
+if (previousTodosJSON !== []) {
```

Comment:
This should compare to `null`, the value of a property of localStorage that doesn't exist is `null`, so that value is what tells us that there is nothing to replace the default.